src/internal/components/InternalEditor.tsx

```diff
@@ -48,7 +48,9 @@ export const InternalEditor = ({
   themeConfig,
 }: InternalEditorProps) => {
   const [canEdit, setCanEdit] = useState<boolean>(false);
-  const [themeModeActive, setThemeModeActive] = useState<boolean>(false);
+  const [themeModeActive, setThemeModeActive] = useState<boolean>(
+    !!templateMetadata.isThemeMode
```

Comment:
Is the coalesce necessary?

Reply:
the !! is b/c I didn't wanna make isThemeMode required (for now), so it doesn't break sites w/o changes.